pages/common/abduco.md

```diff
@@ -0,0 +1,23 @@
+# abduco
+
+> Terminal session manager.
+
+- List sessions:
+
+`abduco`
+
+- Attach to a session, create if not exists:
+
+`abduco -A {{name}} bash`
+
+- Attach to a session with dvtm, create if not exists:
```

Comment:
What does `dvtm` mean? Also, perhaps this will read better?

```suggestion
- Attach to a session with dvtm, creating it if it doesn't exist:
```

Reply:
http://www.brain-dump.org/projects/dvtm/